Preview 469

frostzzone opened the pull request:
Built on preview 454

Fixed level example, and partially .json error from 454 preview(single comma error now)
Adding more examples/blocks

Credit:
@josetrindade1 -Real big smarts
@mrredo - Big smarts

@retrouser955 - Akinator block and Ticket blocks, some examples
@XL83YT  - Reddit blocks and Snake block, added Games and Reddit categories
@EruptionGuy - Bit of ui changing
@ahqsoftwares - Some blocks
@henpokpok - Some blocks

Don't merge this ._.
Lol preview 469 (https://deploy-preview-469--scratch-for-discord.netlify.app/)

ahqsoftwares reviewed (approved):
its approved

BadPotato1007 commented:
A new user left a comment. This user must be approved by a Netlify team owner before comments can be displayed.

[Approve this user](https://app.netlify.com/teams/androz2091/reviewers)

EruptionGuy commented:
cough cough description so outdated

RedMan13 committed:
hjdgzvhdf

RedMan13 committed:
Merge branch 'test' of https://github.com/RedMan13/uzsgyuzffilfscgicsfusdnindusfopnyudfsfdianiovsdgnhtscdfnuuhcg9psudhusdfgvdhusfouhcgsmodfhgosudfmsuic into test

EruptionGuy committed:
Merge pull request #204 from RedMan13/test

goofy ahh machine

JeremyGamer13 committed:
poo

MonacoGuy committed:
Slowmode block

retrouser955 committed:
Merge pull request #205 from MonacoGuy/test

Slowmode block

ChsyDev committed:
Update bot_startup_time.js

ChsyDev committed:
Update bot_startup_time.js

ChsyDev committed:
Update set_bot_game.js

ChsyDev committed:
Update set_bot_stream.js

ChsyDev committed:
Update set_bot_stream.js

ChsyDev committed:
💫 Added a tooltip

mrredo committed:
Fixed since discord.js updated their things

mrredo committed:
Update m_options.js

MonacoGuy committed:
Update toolbox.js

EruptionGuy committed:
Merge pull request #206 from MonacoGuy/test

Removing AHQ Blocks send embed jose/redo

EruptionGuy committed:
New thread blocks

EruptionGuy committed:
Merge branch 'frostzzone:test' into test

ChsyDev committed:
Added in my social because why not 💅

ChsyDev committed:
Update Credit.vue

ChsyDev committed:
extra comma

ChsyDev committed:
Cwheesy Branch 💀

ChsyDev committed:
💫 Added a tooltip

ChsyDev committed:
💫 Added a tooltip

ChsyDev committed:
💫 Added a tooltip

ChsyDev committed:
💫 Added a tooltip

ChsyDev committed:
💫 Added a tooltip because I have nothing elsetodo

ChsyDev committed:
💫 Added a tooltip (send help)

mrredo committed:
Update m_options.js

EruptionGuy committed:
Major thread block updates

EruptionGuy committed:
Merge branch 'frostzzone:test' into test

EruptionGuy committed:
Merge pull request #207 from EruptionGuy/test

Thread blocks revamp!

EruptionGuy committed:
Updated changelog

Commit:
Merge remote-tracking branch 'origin/test' into test

EruptionGuy committed:
Merge pull request #208 from EruptionGuy/test

Updated changelog & Stuff

JeremyGamer13 committed:
changelog blockly displays

JeremyGamer13 committed:
text only theme

EruptionGuy committed:
Changelog update v1

JeremyGamer13 committed:
Merge branch 'test' into test

JeremyGamer13 committed:
Merge pull request #209 from EruptionGuy/test

Changelog update v1

EruptionGuy committed:
Changelog v2

EruptionGuy committed:
Changelog v2

EruptionGuy committed:
Merge pull request #210 from EruptionGuy/test

Changelog v2

RedMan13 committed:
fixed the simple embeds

RedMan13 committed:
Merge branch 'test' of https://github.com/RedMan13/uzsgyuzffilfscgicsfusdnindusfopnyudfsfdianiovsdgnhtscdfnuuhcg9psudhusdfgvdhusfouhcgsmodfhgosudfmsuic into test

EruptionGuy committed:
Merge pull request #211 from RedMan13/test

creme betweens

JeremyGamer13 committed:
switch case update

JeremyGamer13 committed:
forite

waw

JeremyGamer13 committed:
website bloc wowow

EruptionGuy committed:
Revamped user examples menu + other bad stuff

EruptionGuy committed:
jumpscare :trollface:

EruptionGuy committed:
Added "maxlength" attribute to inputs to prevent spamming

EruptionGuy committed:
Updated changelog & removed jumpscare

JeremyGamer13 committed:
allow 404 pages in website blok

EruptionGuy committed:
.

MonacoGuy committed:
Slash block update (by redman13)

MonacoGuy committed:
Merge branch 'frostzzone:test' into test

Commit:
Merge pull request #212 from MonacoGuy/test

slash block update by redman13

ChsyDev committed:
Update set_bot_game.js

JeremyGamer13 committed:
Merge pull request #213 from ChsyDev/patch-1

Update set_bot_game.js

ChsyDev committed:
Update Credit.vue

ChsyDev committed:
Update socials.vue

JeremyGamer13 committed:
Merge branch 'test' into test

JeremyGamer13 committed:
Merge pull request #214 from ChsyDev/test

More tool tips!

JeremyGamer13 committed:
]

JeremyGamer13 committed:
user examples update

JeremyGamer13 committed:
woopsie

JeremyGamer13 committed:
user example fixes & improvements

EruptionGuy committed:
Improved user example once again

EruptionGuy committed:
didn't mean to do this

Commit:
ball hair

EruptionGuy committed:
Fixed thread blocks to be more compatible with others

JeremyGamer13 committed:
some nice appearance update on user examples

JeremyGamer13 committed:
more user example stuff YAYAYYAYAY

JeremyGamer13 committed:
add extra thing to the share link thingy

JeremyGamer13 committed:
[

JeremyGamer13 committed:
fix likes % on share link

EruptionGuy committed:
e

EruptionGuy committed:
Madlick is making sad 😢

EruptionGuy committed:
New font + Deleted cringe background!!!

EruptionGuy committed:
Deleted letter-spacing

Commit:
Thicc

EruptionGuy committed:
Polishing S4D

ChsyDev committed:
💅 Best tooltip ever.

JeremyGamer13 committed:
Merge pull request #215 from ChsyDev/patch-2

💅 Best tooltip ever.

EruptionGuy committed:
Thread block fixes v3;
Improved 'Export to JS' UI

EruptionGuy committed:
Better responsiveness

EruptionGuy committed:
Made code more readable by reducing transparency

Commit:
Create TropicGalxy

JeremyGamer13 committed:
Merge pull request #216 from TropicGalxy/patch-9

Create TropicGalxy

AlexCdDg2 committed:
create stickers category

AlexCdDg2 committed:
e

AlexCdDg2 committed:
e

AlexCdDg2 committed:
e

AlexCdDg2 committed:
e

AlexCdDg2 committed:
e

AlexCdDg2 committed:
e

AlexCdDg2 committed:
e2

AlexCdDg2 committed:
e

AlexCdDg2 committed:
fix lmao

EruptionGuy committed:
Merge pull request #217 from AlexCdDg2/test

sticker blocks ha

JeremyGamer13 committed:
S4D FORUMS YAYAYYAYAY

mrredo please do not break fuckin everyhing i will shit your pants

JeremyGamer13 committed:
fix forum post share link

JeremyGamer13 committed:
noxy wanted me to shove forums in your face!!!

AlexCdDg2 committed:
bug fix

AlexCdDg2 committed:
exis

JeremyGamer13 committed:
Merge pull request #218 from AlexCdDg2/test

some fixes e

JeremyGamer13 committed:
forums ACCOUNT update

JeremyGamer13 committed:
i foror :skull:

ItsLimeNade committed:
Smol update

JeremyGamer13 committed:
Merge pull request #219 from ItsLimeNade/test

Smol update

JeremyGamer13 committed:
fix forums urls on newlines

JeremyGamer13 committed:
yo mom xd

JeremyGamer13 committed:
oops

JeremyGamer13 committed:
yuor mother

JeremyGamer13 committed:
switch case stuff

JeremyGamer13 committed:
forums 1.2

JeremyGamer13 committed:
forum 1.3 yeay

Commit:
Update Delete.js

Commit:
Update Get All.js

Commit:
Update Get.js

Commit:
Update Set To.js

mrredo committed:
who the hell added user member type

mrredo committed:
Update options.js

RedMan13 committed:
yesnt (good for now)

mrredo committed:
Update m_options.js

mrredo committed:
Merge pull request #221 from RedMan13/test

yesnt (good for now)

RedMan13 committed:
general

clean up
should fix a few bugs

RedMan13 committed:
Merge branch 'test' of https://github.com/RedMan13/uzsgyuzffilfscgicsfusdnindusfopnyudfsfdianiovsdgnhtscdfnuuhcg9psudhusdfgvdhusfouhcgsmodfhgosudfmsuic into test

retrouser955 committed:
Merge pull request #222 from RedMan13/test

yesyes

RedMan13 committed:
lenovo emkay 2-13

alot

EruptionGuy committed:
Merge pull request #223 from RedMan13/test

lenovo emkay 2-13

EruptionGuy committed:
Revert "lenovo emkay 2-13"

EruptionGuy committed:
Merge pull request #224 from frostzzone/revert-223-test

Revert "lenovo emkay 2-13"

JeremyGamer13 committed:
Revert "Revert "lenovo emkay 2-13""

This reverts commit 24bd1868fc9a1f2677615b6e10cbdf4dc6e6aeb5.

RedMan13 committed:
added a thing for yes

fixed some tool tips and added a block for db stuff

RedMan13 committed:
added a thing for yes (#225)

RedMan13 committed:
Merge branch 'frostzzone:test' into test

RedMan13 committed:
added bypass and fixed options

and some stuff other

JeremyGamer13 committed:
Merge pull request #226 from RedMan13/test

Title

RedMan13 committed:
i foror

mrredo committed:
user option can not have user and member output

EruptionGuy committed:
Last change to embed

EruptionGuy committed:
Rename

JeremyGamer13 committed:
checkbox mutator module

MonacoGuy committed:
Update socials.vue

EruptionGuy committed:
Merge pull request #228 from MonacoGuy/test

Update socials.vue

MonacoGuy committed:
Update socials.vue

EruptionGuy committed:
Merge pull request #229 from MonacoGuy/test

Update socials.vue

EruptionGuy committed:
Changed it to var to avoid old projects breaking

RedMan13 committed:
Merge branch 'frostzzone:test' into test

RedMan13 committed:
oh ig i never pushed this

RedMan13 committed:
Merge branch 'test' of https://github.com/RedMan13/uzsgyuzffilfscgicsfusdnindusfopnyudfsfdianiovsdgnhtscdfnuuhcg9psudhusdfgvdhusfouhcgsmodfhgosudfmsuic into test

RedMan13 committed:
big poopoo brain moment

RedMan13 committed:
AHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH

RedMan13 committed:
Merge pull request #227 from RedMan13/test

stuff is goofy AHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH

JeremyGamer13 committed:
blockly module stuff

JeremyGamer13 committed:
update examples dropdown & remove toolbox options

JeremyGamer13 committed:
this aint no mf D14081

EruptionGuy committed:
Embed changes

MonacoGuy committed:
Update socials.vue

MonacoGuy committed:
Merge branch 'frostzzone:test' into test

JeremyGamer13 committed:
Merge pull request #230 from MonacoGuy/test

Socials Changes

JeremyGamer13 committed:
edit & shortcut menu updates

retrouser955 committed:
Fixed changes between djs v14 and 13

JeremyGamer13 committed:
dev tools

JeremyGamer13 committed:
i forot

JeremyGamer13 committed:
allow text selection in menus & add debug stuff

JeremyGamer13 committed:
debug menu: finish workspace functions

JeremyGamer13 committed:
fixed menu elements going off of the menu

JeremyGamer13 committed:
debug menu: added APIs menu

JeremyGamer13 committed:
debug menu: remove master api because its not done

noxyyk committed:
Update changelog.vue

EruptionGuy committed:
Merge pull request #231 from noxyyk/patch-15

updated changelog

EruptionGuy committed:
Fixed @noxy 's thing

JeremyGamer13 committed:
debug menu: add refresh for localforage

JeremyGamer13 committed:
debug menu: main workspace check & toolbox menu

JeremyGamer13 committed:
change blocklyModule close menu animation

JeremyGamer13 committed:
same as earlier but for minimized dont animate

JeremyGamer13 committed:
whoopps

JeremyGamer13 committed:
svg path editing stuff

JeremyGamer13 committed:
more debug menu stuff

RedMan13 committed:
dngfasjdghafjkghsdkfjagsdja

uncompletey

JeremyGamer13 committed:
another method to open s4d dev tools

JeremyGamer13 committed:
debug handling changed, credits updated

JeremyGamer13 committed:
amopng us baslls :scream:

RedMan13 committed:
works but doesnt actually

RedMan13 committed:
Merge branch 'frostzzone:test' into test

JeremyGamer13 committed:
custom blocks (load only rn) & other stuff

RedMan13 committed:
FINNALY

IT FINNALY WORKS
AHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHh

RedMan13 committed:
Merge branch 'test' of https://github.com/RedMan13/uzsgyuzffilfscgicsfusdnindusfopnyudfsfdianiovsdgnhtscdfnuuhcg9psudhusdfgvdhusfouhcgsmodfhgosudfmsuic into test

RedMan13 committed:
Merge branch 'test' of https://github.com/RedMan13/uzsgyuzffilfscgicsfusdnindusfopnyudfsfdianiovsdgnhtscdfnuuhcg9psudhusdfgvdhusfouhcgsmodfhgosudfmsuic into test

RedMan13 committed:
r

RedMan13 committed:
fixed?

JeremyGamer13 committed:
Merge pull request #232 from RedMan13/test

ghghghghghghghghghghgh

RedMan13 committed:
yayayayayayayayayayayayayayayayayayayayayayayayayayyayayayayayayayayayyaayyayayayayayayayayyayayayayayayayayayayayyayayayayayayayayayyayayyayayayayayayyayayayayayayayayayyayayyayayayayay

RedMan13 committed:
Merge pull request #233 from RedMan13/test

yayayayayayayayayayayayayayayayayayayayayayayayayayyayayayayayayayaya…

le0o0oo committed:
added tic tac toe game with slash

Commit:
Fixed tic tac toe with slash

Commit:
added placeholders in tic tac toe game

RedMan13 committed:
made stuff and moved stuff (as alwatys)

RedMan13 committed:
bru

RedMan13 committed:
Merge pull request #234 from RedMan13/test

made stuff and moved stuff (as alwatys)

JeremyGamer13 committed:
custom blocks test phase (damn this a big update)

Commit:
Added dashboard block

Commit:
Added the possibility to use text fields

le0o0oo committed:
Merge branch 'frostzzone:test' into main

Commit:
better inputs

Commit:
better dscription

Commit:
added boolean

Commit:
fixed dropdown input

Commit:
added color input

Commit:
removed experimental category

Commit:
Improved "create new dashboard"

Commit:
NO I DID A MISTAK AOIAIOJAOJJ

Commit:
fixed

Commit:
Added commands list page

RedMan13 committed:
farfgamer: added application tools

added dashboard blocks under websites>dashboard
added chart creation under aplications>charts
added slash support to aplications>message games

le0o0oo committed:
Fixed Dashboard blocks (#238)

farf

le0o0oo committed:
Added cookies to the dashboard (#239)

ver cool farf

Commit:
Fixed Tic Tac Toe game from slash breaking all the editor

mrredo committed:
Merge pull request #240 from windows-ubuntu/main

Fixed Tic Tac Toe game from slash breaking all the editor

RedMan13 committed:
added

RedMan13 committed:
added yo mama to s4d

added

JeremyGamer13 committed:
everything "important" from Aug 5th, 2022 to now

JeremyGamer13 committed:
Merge branch 'test' of https://github.com/frostzzone/scratch-for-discord into test

JeremyGamer13 committed:
Revert "Merge branch 'test' of https://github.com/frostzzone/scratch-for-discord into test"

This reverts commit 7f0bd108ac59a791129c72efa4bbd2d949552642, reversing
changes made to 4a33feac6f1b480784d4bbb570933ba290a051bb.

JeremyGamer13 committed:
Revert "everything "important" from Aug 5th, 2022 to now"

This reverts commit 4a33feac6f1b480784d4bbb570933ba290a051bb.

JeremyGamer13 committed:
the slightest change possible

RedMan13 committed:
uh

dont look at this its cRaszy

RedMan13 committed:
just smilo

uh
